spec/cc/engine/analyzers/sexp_lines_spec.rb

```diff
@@ -18,9 +18,9 @@ module CC::Engine::Analyzers
 
         expect(locations.count).to eq 2
         expect(locations[0].begin_line).to eq(3)
-        expect(locations[0].end_line).to eq(7)
+        expect(locations[0].end_line).to eq(4)
```

Comment:
I _think_ this is a product of some tweaks to the way the parser handles these situations.